python_modules/dagster/dagster/core/execution/plan/execute_plan.py

```diff
@@ -164,7 +164,9 @@ def _trigger_hook(
             yield DagsterEvent.hook_completed(step_context, hook_def)
 
 
-def _dagster_event_sequence_for_step(step_context: StepExecutionContext) -> Iterator[DagsterEvent]:
+def _dagster_event_sequence_for_step(
+    step_context: StepExecutionContext, force_local_execution: bool = False
+) -> Iterator[DagsterEvent]:
```

Comment:
i don't love this bool setup - wonder if theres a cleaner refactor. Not blocking, consider for a follow up. 

At least leave a comment explaining the reason at this time for this bool